simple/src/main/java/org/apache/commons/rdf/simple/experimental/AbstractRDFParser.java

```diff
@@ -200,7 +200,7 @@
 	}
 
 
-	private Optional<RDFTermFactory> rdfTermFactory = Optional.empty();
```

Comment:
The use of optional here as a field type makes it impossible to serialise. Need to have the raw values stored in fields if you want to support serialisation in the future, which should otherwise be possible.

Even if it sounds strange, I have had requests about this in the past so people do use serialisation functionality if it is available.